index.html

```diff
@@ -1401,9 +1401,47 @@ <h2>Core Vocabulary Definitions</h2>
 <p>
      <span class="rfc2119-assertion" id="td-context-default-language">
      In the <code>@context</code> array, the optional default language that is 
-     used throughout a TD document is defined within an object as a member with 
+     used throughout a TD document for human-readable strings outside of a <a href="#multilanguage">MultiLanguage</a> object
+     is also defined within an object as a member with 
      a key <code>@language</code> whose value MUST be a string representing a [[!BCP47]] 
      language code.</span>
+     <span class="rfc2119-assertion" id="td-context-default-language-last">
+     The object used to define the default language MUST be the last object in
+     the `@context` array.
+     </span>
+     <span class="rfc2119-assertion" id="td-context-default-language-unique">
+     There MUST be only one object defining a default language in an `@context` array.
+     </span>
+     <span class="rfc2119-assertion" id="td-context-default-language-separate">
+     The object used to define the default language MUST NOT be combined with any other
+     objects defining context extensions (additional vocabulary terms).
+     </span>
+</p>
+<p>
+     Identification of the base direction of human-readable text in TDs 
+     is based on the following set of rules:
+     <ul>
+     <li><span class="rfc2119-assertion" id="td-context-default-language-direction-inference">
+     The base text direction MUST be inferred from the default language.
+     </span></li>
+     <li><span class="rfc2119-assertion" id="td-context-default-language-direction-inference">
+     In cases where a language can be written in more than one script which use 
+     different writing directions, the value used for `@language` MUST
+     include a script subtag so that an appropriate base direction can be inferred. 
+     </span></li>
+     <li><span class="rfc2119-assertion" id="td-context-default-language-direction-default">
+     If no default language is given, the base text direction MUST be assumed to be LTR (left-to-right).
+     </span>  This implies that if the language used in human readable text uses a script
+     that is written RTL (right-to-left), the default language needs to be explicitly specified.</li>
```

Comment:
the default language, _potentially including a script subtag_, needs to be ...

Reply:
OK.

Reply:
Phrase added.